src/js/view/controls/settings-menu.js

```diff
@@ -168,12 +182,7 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
     model.on('change:visualQuality', (changedModel, quality) => {
         const qualitySubMenu = settingsMenu.getSubmenu('quality');
         if (qualitySubMenu) {
-            const items = qualitySubMenu.getItems();
-            const item = items[0].element().querySelector('.jw-auto-label');
-            const levels = model.get('levels');
-            const { mode, level } = quality;
-
-            item.textContent = mode === 'auto' ? `${levels[level.index].label}` : ``;
+            changeAutoLabel(quality, qualitySubMenu, 0);
```

Comment:
Pass the currentLevel here, not `0`.
const currentLevel = model.get('currentLevel');